### ISSUE TITLE
Training with smaller batch size

### ISSUE THREAD

**Issue by mrgloom** [jocicmarko/ultrasound-nerve-segmentation#63]
I figure out that with batch size = 8 (original is 32) network doesn't converges, is it possible to train this Unet implementation with smaller batch size? (i.e. with batch size = 1?)

**Comment by jocicmarko** [comment]
Try adding BatchNormalization layer, it may help you to train with small batch sizes.
